scripts/gulp/tasks/screener.ts

```diff
@@ -14,16 +15,35 @@ task('screener:runner', cb => {
   // screener-runner doesn't allow to pass custom options
   if (argv.filter) process.env.SCREENER_FILTER = argv.filter as string;
 
+  const docsPackageName = '@fluentui/docs';
+
+  const changedPackages = getChangedPackages();
+  const packageInfos = getAllPackageInfo();
+  if (Object.values(packageInfos).every(packageInfo => packageInfo.packageJson.name !== docsPackageName)) {
+    throw new Error(`package ${docsPackageName} does not exist in the repo`);
+  }
+
+  const screenerConfigPath = paths.base('scripts/screener/screener.config.js');
+
   // kill the server when done
-  screenerRunner(paths.base('scripts/screener/screener.config.js'))
-    .then(() => {
-      cb();
-      process.exit(0);
-    })
-    .catch(err => {
-      cb(err);
-      process.exit(1);
-    });
+  const handlePromiseExit = promise =>
+    promise
+      .then(() => {
+        cb();
+        process.exit(0);
+      })
```

Comment:
I don't think it's necessary to explicitly exit 0? If so, can you just do this?
```suggestion
      .then(cb)
```

Reply:
I'm not entirely sure why this was done for gulp tbh.... it was the previous behaviour and I don't want to refactor this now

Reply:
Gulp doesn't handle exit codes properly as you can see here gulpjs/gulp#2081

Reply:
Ah I hadn't noticed that this was the old behavior. Thanks for the background.